src/wallet/rpcwallet.cpp

```diff
@@ -1385,6 +1386,9 @@ void ListTransactions(const CWalletTx& wtx, const string& strAccount, int nMinDe
             if (fLong)
                 WalletTxToJSON(wtx, entry);
             entry.push_back(Pair("size", static_cast<CTransaction>(wtx).GetSerializeSize(SER_NETWORK, PROTOCOL_VERSION)));
+            if(!address.empty() && address!="*"){
```

Comment:
Please conform the rest of the file and move curly bracket to newline

Reply:
Done, thank you.